sql/table.cc

```diff
@@ -5850,6 +5851,62 @@ void TABLE::reset_item_list(List<Item> *item_list, uint skip) const
   }
 }
 
+TABLE_LIST::TABLE_LIST(THD *thd,
+                       LEX_CSTRING _db,
+                       bool _fqtn,
+                       LEX_CSTRING _alias_str,
+                       LEX_CSTRING *_alias,
+                       Table_ident *_table,
+                       thr_lock_type _lock_type,
+                       enum_mdl_type _mdl_type,
+                       ulong _table_options,
+                       bool _info_schema,
+                       st_select_lex *_select_lex,
+                       List<Index_hint> *_index_hints_org,
+                       LEX_STRING *_option)
+{
+  auto *ptr= this;
+  ptr->db= _db;
```

Comment:
you don't really need this `ptr->` after you've copied the code here. It would look more natural without